### PR TITLE
LPS-145029 Audit and refactor tests for component Web Content Display | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-145029

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro [hunk 1]
```diff
@@ -571,6 +571,21 @@ definition {
 		}
 	}
 
+	macro viewMenuItemNotPresent {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro:574]: viewNoEditWC

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro [hunk 2]
```diff
@@ -571,6 +571,21 @@ definition {
...
+		if (IsElementPresent(key_webContentTitle = "${webContentTitle}", locator1 = "WCD#ARTICLE_ELLIPSIS_ICON")) {
+			WebContentDisplayPortletNavigator.gotoArticleEllipsisMenu(webContentTitle = "${webContentTitle}");
+
+			for (var menuItem : list "${menuItemList}") {
+				MenuItem.viewNotPresent(menuItem = "${menuItem}");
+			}
+		}
+		else {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro:582]: Unneeded

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro [hunk 3]
```diff
@@ -623,4 +623,33 @@ definition {
 		}
 	}
 
+	macro viewWithWorkflow {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro:626]: viewWorkflow

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao, I've pushed two extra commits with the changes you requested. Would you have a look at? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro (hunk 2 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro]: Good catch！

**Comment by Tim-Cao** [comment]
@ozysouza
See changes https://github.com/liferay-echo/liferay-portal/pull/7473/commits/ba8a31ae87bee44ee9b4b97bad8e1f220ab89a77

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**